x-pack/plugins/apm/server/lib/services/get_services/get_service_transaction_stats.ts

```diff
@@ -98,10 +98,14 @@ export async function getServiceTransactionStats({
                       missing: '',
                     },
                   },
-                  agentName: {
-                    top_hits: {
-                      docvalue_fields: [AGENT_NAME] as const,
-                      size: 1,
+                  sample: {
+                    top_metrics: {
+                      metrics: {
+                        field: AGENT_NAME,
+                      } as const,
```

Comment:
super tiny nit: Might be easier to read on one line
```suggestion
                      metrics: { field: AGENT_NAME } as const,
```